org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java

```diff
@@ -35,26 +32,29 @@ public void setLoaderService(ILoaderService loaderService) {
 
 	private static XcenServices current;
 
-	public void start(ComponentContext context) {
-		current = this;
+	public void  start(ComponentContext context) {
+		XcenServices.current = this;
 	}
 
 	public void stop() {
-		current = null;
+		XcenServices.current = null;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)